src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx

```diff
@@ -349,23 +232,13 @@ To add a new configuration file for the log forwarding feature:
 
 2. Create a `logging.yml` configuration file, and add the parameters you need. The `logging.d` directory has various `.yml.example` files you can use as a reference or starting point.
 
-    <Callout variant="tip">
-      If you install the infrastructure agent via [Add data](https://one.newrelic.com/marketplace?state=78678a7f-91c5-ca40-ac55-e6b74a50085c) in the UI, the file `logging.yml` is created automatically.
-    </Callout>
-
 The agent automatically processes new configuration files without having to restart the infrastructure monitoring service. The only exception to this is when configuring a custom Fluent Bit configuration.
 
 ## Log forwarding parameters [#parameters]
 
-The infrastructure log forwarding `.yml` config supports the following parameters:
+You're required to set up a `name` and log source parameter in the log forwarding `.yml` config file. To start, define a `name` of the log or logs you want to forward to New Relic. 
```

Comment:
Feels a bit disjointed to have name in code but log source parameter not. Not sure if the fix is both being in code format or neither.